include/libswiftnav/constants.h

```diff
@@ -34,7 +34,7 @@
 /** The official GPS value of Pi.
  * This is the value used by the CS to curve fit ephemeris parameters and
  * should be used in all ephemeris calculations. */
-#define GPS_PI 3.14159265358979323846
+#define GPS_PI 3.1415926535898
```

Comment:
Yep in the IS-GPS-200H, form section 20.3.3.4.3.2:
![image](https://cloud.githubusercontent.com/assets/1215089/11917231/d10c7742-a752-11e5-9547-232c73933ad6.png)